gapis/api/gles/gles.api

```diff
@@ -572,14 +572,9 @@ ContextID              NextContextID
 map!(u64, ref!Context) Contexts
 ref!Context            CurrentContext
 
-sub ref!Context GetContext() {
-  context := Contexts[$Thread]
-  if context == null {
-    _ = newMsg(SEVERITY_ERROR, new!ERR_NO_CONTEXT_BOUND(thread: $Thread))
-    abort
-  }
-  return context
-}
+// Internal "thread-local" global for quick access the current context.
+// It is set in preMutate function (gles.go) before command is mutated.
+@hidden ref!Context ctx
```

Comment:
Isn't this exactly what `CurrentContext` is?

Reply:
Yes, but I didn't want to go and rename all of references to 'ctx' with 'CurrentContext'.
It is more verbose, and more importantly, it would make one massive diff.
I am happy to do that as a followup.

Reply:
Please add that change to this PR. I'm not happy with two copies of the same thing.
Also - aren't you missing logic to set this in GAPII?